src/ray/raylet/design_docs/task_states.rst

```diff
@@ -0,0 +1,36 @@
+A task can be in one of the following states:
+
+- Placeable: the task is ready to be placed at the node where is going to be
```

Comment:
May want to make `Placeable:` bold (and same for the other states). You can do this with `**Placeable:**`.